.travis.yml

```diff
@@ -51,7 +52,7 @@ jobs:
   include:
     - stage: analyze
       script: ./gradlew sonarqube -S
-      if: env(SONAR_LOGIN) IS present
+      if: env(TRAVIS_SECURE_ENV_VARS) IS true
```

Comment:
`TRAVIS_SECURE_ENV_VARS` is `true` but job isn't triggered.. 

https://travis-ci.org/spotbugs/spotbugs-gradle-plugin/jobs/520105306#L741